diff -up sqlite-src-3071401/configure.ac.no-malloc-usable-size sqlite-src-3071401/configure.ac
--- sqlite-src-3071401/configure.ac.no-malloc-usable-size	2012-10-04 22:49:22.000000000 +0300
+++ sqlite-src-3071401/configure.ac	2012-11-06 15:42:26.600210720 +0200
@@ -127,7 +127,7 @@ AC_CHECK_HEADERS([sys/types.h stdlib.h s
 #########
 # Figure out whether or not we have these functions
 #
-AC_CHECK_FUNCS([usleep fdatasync localtime_r gmtime_r localtime_s utime malloc_usable_size])
+AC_CHECK_FUNCS([usleep fdatasync localtime_r gmtime_r localtime_s utime])
 
 #########
 # By default, we use the amalgamation (this may be changed below...)
diff -up sqlite-src-3071401/configure.no-malloc-usable-size sqlite-src-3071401/configure
--- sqlite-src-3071401/configure.no-malloc-usable-size	2012-10-04 22:49:23.000000000 +0300
+++ sqlite-src-3071401/configure	2012-11-06 15:44:51.278149624 +0200
@@ -10293,7 +10293,7 @@ done
 #########
 # Figure out whether or not we have these functions
 #
-for ac_func in usleep fdatasync localtime_r gmtime_r localtime_s utime malloc_usable_size
+for ac_func in usleep fdatasync localtime_r gmtime_r localtime_s utime
 do :
   as_ac_var=`$as_echo "ac_cv_func_$ac_func" | $as_tr_sh`
 ac_fn_c_check_func "$LINENO" "$ac_func" "$as_ac_var"
