diff -up sqlite-3.6.10/src/parse.y.keywords sqlite-3.6.10/src/parse.y
--- sqlite-3.6.10/src/parse.y.keywords	2009-01-15 16:01:46.000000000 +0200
+++ sqlite-3.6.10/src/parse.y	2009-01-22 21:38:42.000000000 +0200
@@ -183,8 +183,8 @@ id(A) ::= ID(X).         {A = X;}
   ABORT AFTER ANALYZE ASC ATTACH BEFORE BEGIN CASCADE CAST CONFLICT
   DATABASE DEFERRED DESC DETACH EACH END EXCLUSIVE EXPLAIN FAIL FOR
   IGNORE IMMEDIATE INITIALLY INSTEAD LIKE_KW MATCH PLAN
-  QUERY KEY OF OFFSET PRAGMA RAISE REPLACE RESTRICT ROW
-  TEMP TRIGGER VACUUM VIEW VIRTUAL
+  QUERY KEY OF OFFSET PRAGMA RAISE RELEASE REPLACE RESTRICT ROW
+  SAVEPOINT TEMP TRIGGER VACUUM VIEW VIRTUAL
 %ifdef SQLITE_OMIT_COMPOUND_SELECT
   EXCEPT INTERSECT UNION
 %endif SQLITE_OMIT_COMPOUND_SELECT
diff -up sqlite-3.6.10/test/table.test.keywords sqlite-3.6.10/test/table.test
--- sqlite-3.6.10/test/table.test.keywords	2008-11-06 00:38:57.000000000 +0200
+++ sqlite-3.6.10/test/table.test	2009-01-22 21:38:42.000000000 +0200
@@ -314,6 +314,14 @@ do_test table-7.3 {
     SELECT * FROM weird;
   }
 } {desc a asc b key 9 14_vac 0 fuzzy_dog_12 xyz begin hi end y'all}
+do_test table-7.3 {
+  execsql {
+    CREATE TABLE savepoint(release);
+    INSERT INTO savepoint(release) VALUES(10);
+    UPDATE savepoint SET release = 5;
+    SELECT release FROM savepoint;
+  }
+} {5}
 
 # Try out the CREATE TABLE AS syntax
 #
