diff -up sqlite-3.5.9/src/expr.c.remove-temp sqlite-3.5.9/src/expr.c
--- sqlite-3.5.9/src/expr.c.remove-temp	2008-05-13 15:50:50.000000000 +0300
+++ sqlite-3.5.9/src/expr.c	2008-09-22 18:04:39.000000000 +0300
@@ -3428,6 +3428,7 @@ int sqlite3GetTempReg(Parse *pParse){
 }
 void sqlite3ReleaseTempReg(Parse *pParse, int iReg){
   if( iReg && pParse->nTempReg<ArraySize(pParse->aTempReg) ){
+    sqlite3ExprWritableRegister(pParse, iReg, iReg);
     pParse->aTempReg[pParse->nTempReg++] = iReg;
   }
 }
